src/libcollections/dlist.rs

```diff
@@ -402,6 +410,51 @@ impl<T> DList<T> {
     pub fn pop_back(&mut self) -> Option<T> {
         self.pop_back_node().map(|box Node{value, ..}| value)
     }
+
+    /// Splits the list into two at the given index. Returns everything after the given index,
+    /// including the index.
+    ///
+    /// This operation should compute in O(n) time.
```

Comment:
I just followed the convention of some other doc strings in the file that mention asymptotic performance (e.g., https://github.com/rust-lang/rust/pull/20406/files#diff-4103b030c3b97fcdc52e866d1cb300fbR363). I'm indifferent to the phrasing though, so if you insist, I'll change it to "This operation computes in O(n) time."